dendrite-config.yaml

```diff
@@ -0,0 +1,73 @@
+# The config file version format
+version: v0
+
+# The matrix specific config
+matrix:
+    # The name of the server. This is usually the domain name, e.g 'matrix.org', 'localhost'.
+    server_name: "example.com"
+    # The path to the PEM formatted matrix private key.
+    private_key: "/var/dendrite/matrix_key.pem"
```

Comment:
Wouldn't we be better assuming that people will run in a homedir?  Also, that the keys & certs match those created by something like the DO deployment script?